### DISCUSSION TITLE
Hỏi về cách sử dụng

### DISCUSSION THREAD

**Discussion by PhamTrung202** [leduyquang753/uet-graduation-latex#2] [Q&A] [answered]
Cho mình hỏi bạn có thể nói rõ hơn về cách chèn file code LaTex vào file LaTex chính không, mình không biết file đấy ở đâu

**Comment by leduyquang753** [comment, marked as answer]
Bạn chọn download tệp ZIP về rồi bạn giải nén các tệp vào vị trí bạn muốn thôi:

![image](https://github.com/leduyquang753/uet-graduation-latex/assets/26449060/d4de332a-7ec5-4774-81ad-416dc8dd14cb)

**Reply by PhamTrung202** [reply]
thế cái compiler b dùng là gì vậy, tại mình lần đầu vọc vạch LaTex 😁

**Reply by leduyquang753** [reply]
XeTeX bạn nhé, quên mất đề cập.

**Reply by PhamTrung202** [reply]
Hehe thanks b nhiều nhé !!
